src/test/resources/retest/recheck/de.retest.web.it.CenterIT/center-ChromeDriver.open.recheck/retest.xml

```diff
@@ -175,12 +187,16 @@
 						</identifyingAttributes>
 						<attributes>
 							<attributes>
+								<entry>
+									<key>covered</key>
+									<value xsi:type="xsd:string">false</value>
+								</entry>
 								<entry>
 									<key>shown</key>
 									<value xsi:type="xsd:string">true</value>
 								</entry>
 								<entry>
-									<key>tab-index</key>
+									<key>tabindex</key>
```

Comment:
@roesslerj should there have been a migrator as part of #561? Also, why track when there is no value?

Reply:
We discussed that and agreed, that this is not necessary, as it is a fairly small change. Or to be more precise: The necessary work seems to big to add a migrator in recheck for recheck-web as we plan to include more breaking changes (this and pseudo-elements).

Reply:
OK, thx, this answers the first part. Regarding the no value: Do you know if this is common? If true, maybe we want to open issue in recheck for this?

Reply:
I looked at the test and tried to reproduce this, as there is no apparent `tabIndex` specified. The code looks like it should work, defaults are properly specified and read.

The only way I could reproduce this is by `tabIndex=""`, but the test has no indication for that...

Reply:
Hmmm, that's weird. We should look out for this in the future. Thanks a lot for testing it!